perfkitbenchmarker/rackspace/rackspace_machine_types.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module contains constants, baked-in assumptions and utility functions related
+ to Rackspace flavors, aka. machine types."""
```

Comment:
Can you add a bit more info about how this gets used? Looks like this is likely to get out of sync with reality unless it's actively maintained.